python_modules/libraries/dagster-fivetran/dagster_fivetran/asset_defs.py

```diff
@@ -101,6 +103,7 @@ def build_fivetran_assets(
     io_manager_key: Optional[str] = None,
     asset_key_prefix: Optional[Sequence[str]] = None,
     metadata_by_table_name: Optional[Mapping[str, MetadataUserInput]] = None,
+    group_name: Optional[str] = None,
```

Comment:
Would you be able to add an entry to the docstring?

Reply:
Done